app/controllers/base_controller.rb

```diff
@@ -15,4 +15,18 @@ def render_error(error_code, error_details = '')
     render json: Error.error_for(error_code, error_details),
            status: Error.status_for(error_code)
   end
+
+  def paginated_render(query, page, per_page)
+    count = query.count
+    response.headers['records'] = count
+    if page && per_page
+      count = query.count
+      page_count = (count / per_page.to_f).ceil
```

Comment:
What do you think about moving this to another method? I think this logic is making this method more difficult to understand. What's more, maybe we can set `total_pages` to be 1 as default and use mod `%` operator when `count > per_page`.